datasets/ccaligned_multilingual/README.md

```diff
@@ -0,0 +1,169 @@
+```
+
+An instance of `sentences` type:
+
+```
+{'LASER_similarity': 1.2734256982803345, 'Source_Sentence': '>>> PhD Students in 2018', 'Target_Sentence': '>>> လုၵ်ႈႁဵၼ်းၸၼ်ႉ Ph.D ၼႂ်းပီ 2018', 'from_english': True}
+```
+
+### Data Fields
+
+For `documents` type:
+
+- `Domain`: a `string` feature containing the domain.
+- `Source_URL`: a `string` feature containing the source URL.
+- `Source_Content`: a `string` feature containing the content on Source_URL.
+- `Target_URL`: a `string` feature containing the target URL.
+- `Target_Content`: a `string` feature containing the content on Target_URL.
+
+For `sentences` type:
+
+- `Source_Sentence`: a `string` feature containig the source sentence.
+- `Target_Sentence`: a `string` feature containing the target sentence.
+- `LASER_similarity`: a `float32` feature representing the LASER similarity score.
+- `from_english`: a `bool` feature representing whether the source sentence is in English.
+
+
+### Data Splits
+
+[Needs More Information]
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+[Needs More Information]
+
+### Licensing Information
+
+[Needs More Information]
+
+### Citation Information
+
+@inproceedings{elkishky_ccaligned_2020,
+ author = {El-Kishky, Ahmed and Chaudhary, Vishrav and Guzm{\'a}n, Francisco and Koehn, Philipp},
+ booktitle = {Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP 2020)},
+ month = {November},
+ title = {{CCAligned}: A Massive Collection of Cross-lingual Web-Document Pairs},
+ year = {2020}
+ address = "Online",
+ publisher = "Association for Computational Linguistics",
+ url = "https://www.aclweb.org/anthology/2020.emnlp-main.480",
+ doi = "10.18653/v1/2020.emnlp-main.480",
+ pages = "5960--5969"
+}
```

Comment:
````suggestion
```
@inproceedings{elkishky_ccaligned_2020,
 author = {El-Kishky, Ahmed and Chaudhary, Vishrav and Guzm{\'a}n, Francisco and Koehn, Philipp},
 booktitle = {Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP 2020)},
 month = {November},
 title = {{CCAligned}: A Massive Collection of Cross-lingual Web-Document Pairs},
 year = {2020}
 address = "Online",
 publisher = "Association for Computational Linguistics",
 url = "https://www.aclweb.org/anthology/2020.emnlp-main.480",
 doi = "10.18653/v1/2020.emnlp-main.480",
 pages = "5960--5969"
}
```

### Contributions

Thanks to [@gchhablani](https://github.com/gchhablani) for adding this dataset.
````
use code block for citation
+ add new section :D